parallaxinc/solo
# Make project name editable

Makes the project name editable, and after editing, updates the projectData object with the new name.

Setting a default name, etc. can (and should be) be dealt with in issue #115.

To Test:
1.) click the project name in the top-right corner
2.) Make sure the styling changes (blue outline)
3.) edit/change the project name
4.) when you click elsewhere (de-focus the project name), the styling should return to normal.
5.) Choose "edit project details" in the hamburger menu to make sure the new project name persists.
